src/main/java/seedu/club/logic/commands/ChangeTaskStatusCommand.java

```diff
@@ -0,0 +1,111 @@
+package seedu.club.logic.commands;
+
+import static seedu.club.logic.parser.CliSyntax.PREFIX_STATUS;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import org.apache.commons.lang3.text.WordUtils;
+
+import seedu.club.commons.core.Messages;
+import seedu.club.commons.core.index.Index;
+import seedu.club.logic.commands.exceptions.CommandException;
+import seedu.club.logic.commands.exceptions.IllegalExecutionException;
+import seedu.club.model.task.Assignee;
+import seedu.club.model.task.Assignor;
+import seedu.club.model.task.Date;
+import seedu.club.model.task.Description;
+import seedu.club.model.task.Status;
+import seedu.club.model.task.Task;
+import seedu.club.model.task.Time;
+import seedu.club.model.task.exceptions.DuplicateTaskException;
+import seedu.club.model.task.exceptions.TaskNotFoundException;
+
+/**
+ * Edits the status of a existing task in the club book.
+ */
+public class ChangeTaskStatusCommand extends UndoableCommand {
+
+    public static final String COMMAND_WORD = "changestatus";
+    public static final ArrayList<String> COMMAND_ALIASES = new ArrayList<>(
+            Arrays.asList(COMMAND_WORD, "status")
+    );
+    public static final String COMMAND_FORMAT = COMMAND_WORD + " INDEX s/STATUS";
```

Comment:
The prefixes in COMMAND_FORMAT and MESSAGE_USAGE Parameters are different.
Also, can you use the prefix from CLI Syntax instead of hardcoding?

Reply:
Oops :sweat_smile: i'll fix that.